### ISSUE TITLE
MySQL server stops working 

### ISSUE THREAD

**Issue by AdrianBell** [miguelgrinberg/microblog#252]
Microblog app deployed successfully on Google Cloud Platform VM and functioning correctly for several hours. Suddenly mMySQL server stops working. When I try to log in: mysql -u root -p I get the following error:

ERROR 2002 (HY000): Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)
--

I could log in before and everything was working. I made no changes and this happened.

**Comment by miguelgrinberg** [comment]
I'm not sure how I can help you. You aren't following the tutorial for this, so there is no reference instructions. Clearly your MySQL server is not running anymore, but I cannot tell you anything besides that.

**Comment by AdrianBell** [comment]
Hi Miguelgrinberg,

Many thanks for getting back to me. I purchased your course and followed
every step up until the end of chapter 17. I deployed the app on a Google
Cloud VM using Ubuntu 16.04. I set it up using MySQL as you described
clearly. Everything was working perfectly for a few hours - registering new
users, writing posts etc. then MySQL stopped working. I didn't change any
configuration files after I completed chapter 17. To make sure I did
everything correctly I destroyed the VM and started from scratch, following
all your instructions but the same issue happened again.

I concluded it was something to do with mysql as I now can't log in using
root user. I just don't understand why it stopped working inexplicably.

Regards,

Adrian
